shell/platform/android/io/flutter/embedding/engine/renderer/SurfaceTextureWrapper.java

```diff
@@ -20,10 +21,17 @@
   private SurfaceTexture surfaceTexture;
   private boolean released;
   private boolean attached;
+  private Runnable onFrameConsumed;
 
   public SurfaceTextureWrapper(@NonNull SurfaceTexture surfaceTexture) {
+    this(surfaceTexture, null);
+  }
+
+  public SurfaceTextureWrapper(
+      @NonNull SurfaceTexture surfaceTexture, @Nullable Runnable onFrameConsumed) {
```

Comment:
java doc for this constructor as well as the previous one to document when to use which